frontend/src/pages/importVault/ImportVaultView.tsx

```diff
@@ -88,7 +86,7 @@ const ImportVaultView = () => {
           new TextDecoder('utf-8', { fatal: true }).decode(uint8Array);
           setDecryptedVaultContent(uint8Array);
           setContinue(true);
-        } catch (e) {
+        } catch {
```

Comment:
![CleanShot 2024-11-28 at 15 29 16@2x](https://github.com/user-attachments/assets/2a39a3b5-160e-49cb-bf4c-0eb5afbeb8bf)

The logic is wrong somewhere , I can't import a vault